### PR TITLE
Feature/charge payment status

### PR DESCRIPTION
**Pull request by meletios**

### CODE REVIEW

File: payment-service/src/main/resources/schema.graphqls [hunk 1]
```diff
@@ -8,7 +8,9 @@ type Query {
   chargeDate: LocalDate! @deprecated(reason: "Use `nextChargeDate`")
   nextChargeDate: LocalDate
   registerAccountProcessingStatus : RegisterAccountProcessingStatus! @deprecated(reason: "Use `directDebitStatus`")
-  directDebitStatus: DirectDebitStatus!
+  directDebitStatus: DirectDebitStatus!  @deprecated(reason: "Use `chargeablePaymentMethodStatus`")
```
Comment (sampettersson) [inline payment-service/src/main/resources/schema.graphqls:11]: *Use `payinMethodStatus`

File: payment-service/src/main/java/com/hedvig/paymentservice/services/adyen/extentions/PaymentsResponseExtentions.kt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+package com.hedvig.paymentservice.services.adyen.extentions
```
Comment (FredrikAreschoug) [inline payment-service/src/main/java/com/hedvig/paymentservice/services/adyen/extentions/PaymentsResponseExtentions.kt:1]: :dagger:

File: payment-service/src/main/java/com/hedvig/paymentservice/services/adyen/dtos/AdyenPaymentsResponse.kt [hunk 3]
```diff
@@ -1,14 +1,51 @@
...
+      CHALLENGESHOPPER,
+      IDENTIFYSHOPPER,
+      PENDING,
+      RECEIVED,
+      REDIRECTSHOPPER,
+      PARTIALLYAUTHORISED,
+      PRESENTTOSHOPPER,
+      UNKNOWN -> {
+        PaymentResponseResultCode.PENDING
+      }
```
Comment (FredrikAreschoug) [inline payment-service/src/main/java/com/hedvig/paymentservice/services/adyen/dtos/AdyenPaymentsResponse.kt:39]: can remove the `{ }`

File: payment-service/src/main/java/com/hedvig/paymentservice/services/adyen/dtos/AdyenPaymentsResponse.kt [hunk 4]
```diff
@@ -1,14 +1,51 @@
...
+
+      ERROR,
+      CANCELLED,
+      REFUSED -> {
+        PaymentResponseResultCode.FAILED
+      }
```
Comment (FredrikAreschoug) [inline payment-service/src/main/java/com/hedvig/paymentservice/services/adyen/dtos/AdyenPaymentsResponse.kt:45]: Same here

**Pull request merged**